new-frontend/src/app/pages/task/create/task-create.component.ts

```diff
@@ -133,15 +131,65 @@ export class TaskCreateComponent implements OnInit, OnDestroy {
     }
   }
 
+  async handleFirstPrepocessor(isFirst: boolean): Promise<void> {
+    if (!isFirst) return;
+    this.isLoadingColumns = true;
+
+    // collect data to collect columns from database
+    const taskDatabases: TaskDatabase[] = this.getSelectedDatabases();
+    // TODO modify when choosing database for preprocessing is implemented
+    const taskDatabase = taskDatabases[0];
+
+    // collect a node that is online
+    const node = await this.getOnlineNode();
```

Comment:
Shouldn't this be done on the selected node? Or at least first on the selected node, otherwise on a random online node.